apps/network-alerts/test/unit/notification/notification_test.js

```diff
@@ -34,7 +34,8 @@ suite('Network Alerts - Notification handling', function() {
     var message = {
       title: 'Some title',
       body: 'Some body',
-      clicked: true
+      clicked: true,
```

Comment:
This only nit is left :) Since you don't use ```Utils``` inside ```notification.js``` you don't need __anything__ related to ```Utils``` in test as well (in jshint global, in ```require(....mock_util)```, no need in ```MocksHelper``` since it has been used only to mock ```Utils``` and ```this.sinon.stub(Utils, 'parseParams')``` as well )